### DISCUSSION TITLE
More convenient Saving

### DISCUSSION THREAD

**Discussion by starush** [Sanster/IOPaint#219] [Ideas]
Lama Cleaner is a beautiful tool that allows quick cleaning. However, when work is finished, you have to save the result. And this is the terrible part. In most image editors it's just supposed to save at the same location, the same folder where original image has been open from, and using the same name. Please, please don't use "browser default" save dialog.  Please make it more convenient. Put it on hotkey, supply the option to overwrite by default. Thank you in advance!

**Comment by Sanster** [comment]
Opened an issue here: https://github.com/Sanster/lama-cleaner/issues/220

**Comment by starush** [comment]
Maybe Electron browser components may become a solution? I have read that Electron allows to know a full local path of file.

**Reply by Sanster** [reply]
Yes, Electron can access the local file system. Currently working on a macOS m1 version desktop application, I think the `overwrite` feature is a good feature to be added to this version.
